facebook/__init__.py

```diff
@@ -235,7 +243,7 @@ def request(
             # or it does not need `access_token`.
             if post_args and "access_token" not in post_args:
                 post_args["access_token"] = self.access_token
-            elif "access_token" not in args:
+            elif args and "access_token" not in args:
```

Comment:
Totally makes sense, removed the change to the request method and added an empty args dict to the call.